tools/tests/py2fgen/wrappers/references/diffusion/diffusion.py

```diff
@@ -0,0 +1,779 @@
+import logging
+from diffusion import ffi
+from icon4py.tools.py2fgen import runtime_config, _runtime, _definitions, _conversion
```

Comment:
Is there any reason to split pure run-time dependencies in two modules: `runtime_config` and `_runtime`? It feels a bit confusing to me.

Reply:
only because one contains the public environment variables, the other is pure internal